gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/pom.xml

```diff
@@ -890,4 +891,60 @@
     <modules>
         <module>gravitee-am-management-api-standalone-distribution-zip</module>
     </modules>
+
```

Comment:
**todo**: Did you forgot to add this profile definition into the gateway standalone distrobution pom file?